src/bindist/autodos7.bat

```diff
@@ -0,0 +1,6 @@
+@ECHO Off
+PATH D:\OPENDOS;D:\EMUBIN;D:\
+VERIFY OFF
+PROMPT [OpenDOS] $P$G
+SET OPENDOSCFG=D:\OPENDOS
+CALL D:\EMUBIN\UXMACROS.BAT
```

Comment:
Of course that is already there and that hides the C:\\> from being shown, but DOS will still process it as an extra line. I agree that text files should always have newlines at the end, but for DOS batch files I make an exception for this reason.